lib/http.js

```diff
@@ -237,14 +241,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
           {httpStatusCode: 403, public: true});
       }
       const {profileAgentId} = req.params;
-      const {capabilitySet} = await capabilitySets.get({profileAgentId});
-      res.json(capabilitySet);
+      const {profileAgent: {zcaps}} = await profileAgents.get(
+        {id: profileAgentId});
+      res.json(zcaps);
     }));
 
   // update profile agent's zcaps (updates their capability set)
+  // FIXME: will this API also handle the removal of zcaps?
```

Comment:
Seems like it to me. @gannan08 was that your thinking?

Reply:
I can't think of a good reason why we would prevent one from using the update API to remove zcaps. It should handle the removal of zcaps.